indexing-hadoop/src/main/java/io/druid/indexer/hadoop/DatasourceInputFormat.java

```diff
@@ -160,7 +161,8 @@ private DatasourceInputSplit toDataSourceSplit(
       locations = getFrequentLocations(segments, fio, conf);
     }
     catch (Exception e) {
-      logger.error("Exception thrown finding location of splits", e);
+      logger.error(e, "Exception thrown finding location of splits");
+      throw Throwables.propagate(e);
```

Comment:
i don't this needs to be propagated. finding locations should only be best effort.